src/Base/Support/Traits/CheckPluginActive.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace OWC\OpenPub\Base\Support\Traits;
+
+trait CheckPluginActive
+{
+    /**
+     * @param string $file // example: 'pdc-internal-products/pdc-internal-products.php'
+     */
+    public function isPluginActive(string $file): bool
+    {
+        if (! function_exists('is_plugin_active')) {
+            include_once ABSPATH . 'wp-admin/includes/plugin.php';
+        }
+
+        return is_plugin_active($file);
+    }
+
+    public function isPluginSeoPressActive(): bool
```

Comment:
Ik zou persoonlijk deze trait geen kennis geven over wat seopress of yoast is en bij Yoast en Seopress `$this->isPluginActive("de plugin")` aanroepen maar dat is mijn mening